eldritch/events.py

```diff
@@ -93,9 +93,18 @@ def log(self, state) -> str:
 
 
 class ChoiceEvent(Event):
+  def __init__(self):
+    super().__init__()
+    self._choices = None
+
+  @property
+  def choices(self):
+    return self._choices
 
   @abc.abstractmethod
-  def resolve(self, state, choice=None) -> NoReturn:  # pylint: disable=arguments-differ
+  def resolve(
+      self, state, choice=None
```

Comment:
I split this line so that I could give the type hint that state is an eldritch.GameState, but then ended up in a circular import purgatory...

Reply:
Leaving this link for my future reference. https://docs.python.org/3/library/typing.html#constant